### ISSUE TITLE
 [Adaptive Card iOS - Input.Number.ErrorMessage.json]: Voice over focus is not landing on keypad after entering into textfield. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8402]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice Over
4. Navigate to "V1.3" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Input.Number.ErrorMessage.json' under  'Elements' list item and activate it.
7. Card will be appear.
8. Navigate to 'Required Input Number' text field and double tap on it.
9. Observe 'voice over' issue.

**Actual Result:**
On double tapping on the **'Required Input.Number'** edit field, voice over focus is not landing on the keypad.

**Expected Result:**
After double tapping on the **'Required Input.Number'** voice over focus should be land on the keypad.

**Notes:**
v1.3-Elements-Issue is repro'ing with Input.Number.Label.json
v1.3-Elements-Issue is repro'ing with Input.Text.ErrorMessage.json
v1.3-Elements-Issue is repro'ing in Input.Text.Label.json

**User Experience:**
Voice Over user will be impacted if Voice over focus is not landing on keypad after entering into textfield.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/227788268-48b3f2d8-38fd-41e0-8cd6-f23d685fb09c.mov

### Card JSON

```JSON
NA
```
